cleanup.yml

```diff
@@ -0,0 +1,10 @@
+---
+# This playbook performs service cleanup
+#
+
+- hosts: localhost
+  connection: local
+  vars_files:
+    - keys.yml
+  tasks:
+    - include: roles/local/tasks/cleanup.yml
```

Comment:
I changed the name of this role so it should now be:

  - include: roles/openstack_infra/tasks/cleanup.yml

Reply:
Oops